mysql/datadog_checks/mysql/config_models/instance.py

```diff
@@ -81,6 +81,24 @@ class Config:
     key: Optional[str]
 
 
+class AdditionalStatus(BaseModel):
```

Comment:
This file is autogenerated. Changes here need to come from the [spec file ](https://github.com/DataDog/integrations-core/blob/master/mysql/assets/configuration/spec.yaml) and then synced with `ddev validate models -s`